FirebaseAdmin/FirebaseAdmin.Tests/Auth/FirebaseAuthTest.cs

```diff
@@ -127,19 +128,34 @@ public void TenantManagerNoProjectId()
         public void ServiceAccountId()
         {
             FirebaseApp.Create(new AppOptions
-                {
-                    Credential = MockCredential,
-                    ServiceAccountId = "test-service-account",
-                });
+            {
+                Credential = MockCredential,
+                ServiceAccountId = "test-service-account",
+            });
 
             var tokenFactory = FirebaseAuth.DefaultInstance.TokenFactory;
 
             Assert.IsType<FixedAccountIAMSigner>(tokenFactory.Signer);
         }
 
-        public void Dispose()
+        public virtual void Dispose()
         {
             FirebaseApp.DeleteAll();
         }
     }
+
+    public class EmulatorFirebaseAuthTest : FirebaseAuthTest, IDisposable
```

Comment:
We don't have any assertions in these tests to verify whether the emulator mode is working or not. Also our implementation only looks up the emulator host env variable when initializing the `FirebaseUserManager` instance. Most of these tests don't go that far into the execution. So they are not really testing the emulator mode.

So my recommendation for now would be to revert changes to this file. In the future we can figure out a way to run our integration test suite against the emulator (we've done something similar in our Node.js repo already). In the meantime we have decent unit test coverage in `FirebaseUserManagerTest` for emulator mode.

Reply:
I want to be clear. I have not done any manual testing either. I have reverted.